.deploy/rafaelrglima/local/settings.json

```diff
@@ -0,0 +1,38 @@
+{
+    "debugModeOn": true,
+    "public": {
+        "dictionary":{
+            "defaultLanguage": "en",
+            "userDefaultLanguage": "es",
+            "googletranslateapikey": "AIzaSyAwsdfsdfdserraY0",
+            "debug": true
+        },
+        "project": {
+            "name": "rafaelrglima System",
+            "abrev": "DS",
+            "url": "localhost:3000"
+        },
+        "base":{
+            "url": "../../"
+        },
+        "os": "linux",
+        "amazon": {
+            "url": "https://s3-us-west-2.amazonaws.com",
+            "bucket": "rafaelrglima-s3",
+            "imageStorage": "localhost"
+        }
+    },
+    "private": {
+        "storage":{
+            "production": "/tmp"
+        },
+        "mailgun":{
+            "apiKey": "key-c9aec0821c458557b26895145f3c4591",
```

Comment:
I hope this is not your actual API key...
Those files (`.deploy/rafaelrglima/*`) looks very personal, and I would assume you haven't put personal credentials in those. (hint, if you did, you want to revoke all of them).

Reply:
@bentiss 

Those are not actual API keys, they have been changed before committing. This PR is just for cherry picking changes related to:

1) Reorganizing directory structure, by moving Wekan dependy repos like wekan-ldap etc to wekan/packages/ directory so that everything can be built just with command `meteor`

2) Having Wekan deployed with `mup`, so there is also Nginx/Let's Encrypt automatically cert renewing Docker containers.

3) Hopefully, fixing bug related to Wekan meteor-1.8 branch version [working with MongoDB 4.x with Snap](https://forum.snapcraft.io/t/how-to-connect-to-localhost-mongodb-in-snap-apparmor-prevents/7793) - that is not tested yet.

4) Updates to Meteor 1.8.x packages.

This PR has been added to this Wekan repo so that's is just easier to me to cherry pick changes, and keep on my TODO list.

Reply:
Oh, OK.

Then I guess sorry for the noise then :)